src/modules/blockchainApplication/connection/components/ConnectionSummary/index.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import Dialog from '@theme/dialog/dialog';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { withRouter } from 'react-router';
+import ValueAndLabel from 'src/modules/transaction/components/TransactionDetails/valueAndLabel';
+import { PrimaryButton, SecondaryButton } from 'src/theme/buttons';
+import { EVENTS } from '@libs/wcm/constants/lifeCycle';
+import { addSearchParamsToUrl } from 'src/utils/searchParams';
+import ConnectionContext from '@libs/wcm/context/connectionContext';
+import useSession from '@libs/wcm/hooks/useSession';
+import BlockchainAppDetailsHeader from '../../../explore/components/BlockchainAppDetailsHeader';
+import AccountsSelector from './AccountsSelector';
+import styles from './connectionSummary.css';
+
+// eslint-disable-next-line max-statements
+const ConnectionSummary = ({ history }) => {
+  const [addresses, setAddresses] = useState([]);
+  const { t } = useTranslation();
+  const { events } = useContext(ConnectionContext);
+  const { approve, reject } = useSession();
+
+  const connectHandler = async () => {
+    const status = await approve(addresses);
+    addSearchParamsToUrl(history, { modal: 'connectionSuccess', status });
+  };
+
+  const rejectHandler = () => {
+    reject();
+  };
```

Comment:
I don't think there is a need for this because only `reject` which is a function was called here

Reply:
Good point.